packages/legend-query-builder/style/_query-builder.scss

```diff
@@ -908,6 +913,10 @@
     height: 100%;
     padding: 2rem;
   }
+
+  .panel-loading-indicator__container {
+    margin-top: -2rem;
```

Comment:
negative `margin` is usually a sign of hack, what's happening here?

Reply:
Re-addressed in panel loading indicator component instead